tests/drizzle.test.ts

```diff
@@ -29,48 +20,79 @@ const testTable = pgTable(testTableName, {
   value: text('value'),
 });
 
+const countersTable = pgTable('counters', {
+  id: serial('id').primaryKey(),
+  name: varchar('name', { length: 255 }),
+  value: integer('value').notNull().default(0),
+});
+
 let insertCount = 0;
 
 @OrmEntities()
 export class NoEntities {}
 
-@OrmEntities({ testTable })
+type DrizzleDB = NodePgDatabase<{ testTable: typeof testTable; counters: typeof countersTable }>;
+const getDrizzleDB = () => DBOS.drizzleClient as DrizzleDB;
+
+@OrmEntities({ testTable, countersDb: countersTable })
 class TestClass {
-  @Transaction()
-  static async testInsert(txnCtxt: TransactionContext<NodePgDatabase>, value: string) {
+  @DBOS.transaction()
+  static async testInsert(value: string) {
     insertCount++;
-    const result = await txnCtxt.client.insert(testTable).values({ value }).returning({ id: testTable.id });
+    const result = await getDrizzleDB().insert(testTable).values({ value }).returning({ id: testTable.id });
     return result[0].id;
   }
 
-  @Transaction()
-  static async testSelect(txnCtxt: TransactionContext<NodePgDatabase>, id: number) {
-    const result = await txnCtxt.client.select().from(testTable).where(eq(testTable.id, id));
+  @DBOS.transaction()
+  static async testSelect(id: number) {
+    const result = await getDrizzleDB().select().from(testTable).where(eq(testTable.id, id));
     return result[0].value;
   }
 
-  @Transaction()
-  static async testQuery(ctx: TransactionContext<NodePgDatabase<{ testTable: typeof testTable }>>) {
-    const result = await ctx.client.query.testTable.findMany();
+  @DBOS.transaction()
+  static async testQuery() {
+    const result = await getDrizzleDB().query.testTable.findMany();
     return result[0].value;
   }
 
   @DBOS.workflow()
   static async testWf(value: string) {
-    const id = await DBOS.invoke(TestClass).testInsert(value);
-    const result = await DBOS.invoke(TestClass).testSelect(id);
+    const id = await TestClass.testInsert(value);
+    const result = await TestClass.testSelect(id);
     return result;
   }
 
-  @Transaction()
-  static async returnVoid(_ctxt: TransactionContext<NodePgDatabase>) {}
+  @DBOS.transaction()
+  static async returnVoid() {}
 
-  @Transaction()
-  static async unsafeInsert(txnCtxt: TransactionContext<NodePgDatabase>, key: number, value: string) {
+  @DBOS.transaction()
+  static async unsafeInsert(key: number, value: string) {
     insertCount++;
-    const result = await txnCtxt.client.insert(testTable).values({ id: key, value }).returning({ id: testTable.id });
+    const result = await getDrizzleDB().insert(testTable).values({ id: key, value }).returning({ id: testTable.id });
     return result[0].id;
   }
+
+  static conflictTrigger: Promise<void> | undefined = undefined;
+
+  @DBOS.transaction({ isolationLevel: IsolationLevel.Serializable })
```

Comment:
I think our default isolation level is already serializable... We can test explicitly setting isolation level somewhere else.